templates/base.html

```diff
@@ -56,22 +56,87 @@ <h1 class="brand m-0 h4"><a href="/">2021 Georgia Early Vote Ballot Tracker</a><
 
     <div id="content">
       {% block intro %}
-      <div class="container pt-5 mb-3">
-        <p class="m-0">
-          On January 5th there will be a statewide runoff election in Georgia. Let's get all of Georgia voting in this
-          important election! <a href="https://www.facebook.com/search/top?q=my%20friends%20who%20live%20in%20georgia" target="_blank" rel="noopener">Check which of your Facebook friends live in Georgia</a>, then use the search box below to look them up
-          and see if they have sent in their mail-in ballots.
-          <br><br>
-          All of this data comes from the <a href="https://elections.sos.ga.gov/Elections/voterabsenteefile.do">publicly
-          available files</a> published by the Georgia Secretary of State. If
-          you are wanting to track your own ballot status, consider utilizing
-          <a href="https://georgia.ballottrax.net/voter/">Georgia's official
-          site for this purpose</a>.
-        </p>
-      </div>
+        <!-- QUESTION: Why does my override for bg-danger not work but btn-danger does? -->
+        <div class="text-center text-white py-3" style="background-color: #e12814;" >
+          <!-- QUESTION: Should we consider the pre-Eday VA countdown ticker here? -->
+          <h1>The Georgia runoff is on January 5th</h1>
+          <h2>This election will determine who controls the U.S. Senate.</h2>
+        </div>
+
+        <div class="container pt-3 mb-3">
+          <div class="card rounded mb-3">
+            <div class="card-body">
+              <h3 class="card-title mt-0">What's happening</h5>
+              <p class="card-text">Runoff elections are all about turnout: we have <strong>XX days</strong> left to make sure every Georgia voter can make their voice heard. And we're behind: by this point in the November election, <strong>{{ stats.returned_general|commafy }} voters</strong> had already voted. But only <strong>{{ stats.returned_special|commafy }} voters</strong> have voted so far in the runoff. Help us catch up!</p>
```

Comment:
The wording here is a little deceptive. I believe `stats.returned_general` will return the TOTAL number of early returned ballots, not the ones that had been returned X days in advance. That said, it is fairly easy to pull the number you are referencing here so I think you can leave as-is and I can help update the stats in a follow-up commit

Reply:
also i should note - we'll probably want to calculate "X days" in advance by simply doing jan 5 - today. BUT the numbers that we report on will be for something like X+1 or X+2 days in advance, because there is some lag in reporting. I think that's probably fine to not mention here but just want to mention it for completeness

Reply:
sure, sounds good 

def up for making any changes. i erred on the side of leaving in all the copy we had before, making some adjustments to the mocks, but not deleting anything (yet)